skbio/metadata/_repr.py

```diff
@@ -126,6 +127,13 @@ def _format_positional_metadata_column(self, key, dtype):
         dtype_fmt = '<dtype: %s>' % str(dtype)
         return self._wrap_text_with_indent(dtype_fmt, key_fmt, 1)
 
+    def _process_interval_metadata(self):
+        if self._obj.has_interval_metadata():
+            self._lines.add_line('Interval metadata:')
+            self._lines.add_line(
+                    self._indent + '%d interval features' % len(
```

Comment:
sure. so the seq repr looks ok to show just the number of interval features as in line 430?